test/parallel/test-fs-symlink-dir-junction.js

```diff
@@ -21,19 +21,17 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
 
 // test creating and reading symbolic link
-const linkData = path.join(common.fixturesDir, 'cycles/');
+const linkData = path.join(fixtures.fixturesDir, 'cycles/');
```

Comment:
This could be using method `fixtures.path` instead of `path.join`.

Reply:
Thank you, great suggestion.

Reply:
I have made the change.